common/src/main/java/org/apache/drill/exec/metrics/DrillCounters.java

```diff
@@ -0,0 +1,53 @@
+package org.apache.drill.exec.metrics;
+
+import com.codahale.metrics.Counter;
+
+public class DrillCounters {
```

Comment:
Please add java doc.

Reply:
Done, please look if anything should be changed.